scripts/suite_objects.py

```diff
@@ -1605,29 +1666,76 @@ def write(self, outfile, errcode, errmsg, indent):
         my_args = self.call_list.call_string(cldicts=cldicts,
                                              is_func_call=True,
                                              subname=self.subroutine_name)
-
+        #
         outfile.write('', indent)
         outfile.write('if ({} == 0) then'.format(errcode), indent)
-
+        #
         # Write debug checks (operating on variables
         # coming from the group's call list)
+        #
+        if self.__var_debug_checks:
+            outfile.write('! ##################################################################', indent+1)
```

Comment:
Could be
```suggestion
            outfile.comment('#'*66, indent+1)
```
You could even make this string a variable in this file or at least the length (for reuse). This comment applies to other uses of this string so not commenting there.

Reply:
I will clean this up in a subsequent PR.